AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -26,11 +26,14 @@
 #include <CGAL/internal/AABB_tree/AABB_search_tree.h>
 #include <CGAL/internal/AABB_tree/Has_nested_type_Shared_data.h>
 #include <CGAL/internal/AABB_tree/Primitive_helper.h>
+#include <CGAL/AABB_traits_construct_by_sorting.h>
```

Comment:
This should be avoided

Reply:
Related to https://github.com/CGAL/cgal/pull/5893#discussion_r688562692, I think a forward declaration would also work, but that's not really any better.